packages/office-ui-fabric-react/src/utilities/grid/GridCell.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import {
+  autobind,
+  BaseComponent,
+  css,
+  getId,
+} from '../../Utilities';
+import { IGridCellProps } from './GridCell.Props';
+import { CommandButton } from '../../Button';
+
+export class GridCell extends React.Component<IGridCellProps, null> {
+
+  public static defaultProps = {
+    cellShape: 'circle',
+    disabled: false
+  };
+
+  private _id: string;
+
+  constructor(props: IGridCellProps) {
+    super(props);
+
+    this._id = props.id || getId('gridCell');
+  }
+
+  @autobind
+  private _onClick() {
+    let {
+        onClick,
+      disabled,
+      item
+      } = this.props;
+
+    if (onClick && !disabled) {
+      onClick(item);
+    }
+  }
+
+  @autobind
+  private _onMouseEnter() {
+    let {
+        onHover,
+      disabled,
+      item
+      } = this.props;
+
+    if (onHover && !disabled) {
+      onHover(item);
+    }
+  }
+
+  @autobind
+  private _onMouseLeave() {
+    let {
+        onHover,
+      disabled,
+      item
+      } = this.props;
+
+    if (onHover && !disabled) {
+      onHover(item);
```

Comment:
you should not be passing the item here. OnMouseLeave should be telling the creator that there is no longer a hover happening (you can also add a comment for this on in the props def